rebuildall.sh

```diff
@@ -371,32 +371,33 @@ then
 fi
 
 prepare_BUILD_file
-
+cpu_resource_limit_param="--local_cpu_resources=HOST_CPUS*.5"
+ram_resource_limit_param="--local_ram_resources=HOST_RAM*.5"
 if [ $no_s3ut_build -eq 0 ]
 then
   bazel build //:s3ut --cxxopt="-std=c++11" --define $MOTR_INC_ \
                       --define $MOTR_LIB_ --define $MOTR_HELPERS_LIB_ \
                       --spawn_strategy=standalone \
-                      --strip=never
+                      --strip=never $cpu_resource_limit_param $ram_resource_limit_param
```

Comment:
you can have that within "" , basically what codacy is complaining

Reply:
yes. will do it